src/hooks/useTokenBalance.ts

```diff
@@ -85,22 +85,25 @@ export const useBurnedBalance = (tokenAddress: string) => {
 }
 
 export const useGetBnbBalance = () => {
+  const [isFetching, setIsFetching] = useState(false)
   const [balance, setBalance] = useState(BIG_ZERO)
   const { account } = useWeb3React()
   const { lastUpdated, setLastUpdated } = useLastUpdated()
 
   useEffect(() => {
     const fetchBalance = async () => {
+      setIsFetching(true)
```

Comment:
I added a fetching state because we don't know if the balance is zero because it is zero or just because it is the fallback value.

Reply:
It could use same `FetchStatus` as `useTokenBalance` above.

Reply:
Oh, missed that! Thanks.